CIP-????/README.md

```diff
@@ -0,0 +1,205 @@
+Sometimes these events have clear triggers (e.g. Plutus script execution)
+and sometimes they provide intermediate calculations performed by the ledger rules
+(e.g. the reweard calculation).
+The events come with zero cost to a running node if not used and are not stored in the ledger state.
+Documentation about the existing events can be found
+[here](https://github.com/input-output-hk/cardano-ledger/blob/master/docs/LedgerEvents.md).
+
+### Soft forks and Hard forks
+
+Since most ledger CIPs will involve backwards incompatible changes,
+the following two definitions are helpful:
+
+**Hard fork** - A *hard fork* is a change to the protocol (not restricted to the ledger)
+resulting in a single new block definition becoming valid.
+
+Alternatively, a hard fork is a backwards incompatible change for both
+block producers and block validators.
+
+**Soft fork** - A *soft fork* is a change to the protocol (not restricted to the ledger)
+resulting in fewer blocks being valid.
+
+Alternatively, a soft fork is a backwards incompatible change for
+block producers, but a backwards compatible change for block validators.
+
+### Serialization
+
+Transactions and blocks are serialized with
+[CBOR](https://www.rfc-editor.org/rfc/rfc7049)
+and specified with
+[CDDL file](https://www.rfc-editor.org/rfc/rfc8610).
+
+Serialization changes to the ledger are discussed in
+[CIP-80](https://github.com/cardano-foundation/CIPs/pull/372).
+
+Note that the serialisation format of the ledger state is unspecified and left as an
+implementation detail (unlike the format of blocks).
+
+### The ledger-script interface
+
+The ledger and Plutus scripts have a common interface, described in [CIP-35].
+CIPs relating to this inteface are relevant to both the ledger and to the Plutus CIP categories.
+
+## Specification
+
+### What merits a ledger CIP?
+
+The criterion for deciding if a change to the ledger merits a CIP is as follows:
+changes to the ledger require going through the CIP process whenever
+every implementation of the Cardano ledger needs to be standardized on the details.
+
+Bug fixes are an exception to this criterion, they do not merit a CIP except in the case
+that the fix is substantially complicated.
+A "bug fix" is a change to behavior where:
+- The implemented behavior does not match the specification; or
+- The specified behavior is clearly wrong (in the judgment of relevant experts)
+
+Serialization changes are another possible exception to the criterion.
+Many serialization changes can be handled as a part of the normal development process
+without the need for a CIP.
+Dramatic changes to the serialization, however, may benefit from the CIP process.
+
+The ledger rules MUST be standardized in order for consensus to be maintained,
+but things like the ledger events are more open to debate.
+
+Changes to the protocol parameter values do not require a CIP since they are
+a governance issue (see [CIP-1694]).
+
+### Ledger Core CIP registry
```

Comment:
I think this isn't needed any more since there's a category that does the job of aggregating things.